bin/blazegraph_load_api.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# On a debian package installation, the original file could be found at:
+#/usr/bin/loadRestAPI.sh
+
+if [ "$1" == "" -o "$1" == "-h" -o "$1" == "--help" ]; then
+    echo "Usage: $0 {file_or_directory} [{graph_namespace}]"
+    exit
+else
+    FILE_OR_DIR=$1
+fi
+
+if [ ! -z "$2" ]; then
+    NAMESPACE=$2
+else
+    NAMESPACE=kb
+fi
+
+[ -f /etc/default/blazegraph ] && source /etc/default/blazegraph
+[ -z "$JETTY_PORT" ] && JETTY_PORT=9999
+
+BG_URL="http://localhost:${JETTY_PORT}/blazegraph/dataloader"
+
+# On a debian package installation, there is a file at
+# /etc/blazegraph/RWStore.properties
+# That file is almost identical to the source file in 
+# BLAZEGRAPH_RELEASE_2_1_4/src/resources/deployment/nss/WEB-INF/RWStore.properties
+# Other installations might put this file in:
+#export NSS_DATALOAD_PROPERTIES=/usr/local/blazegraph/conf/RWStore.properties
+
+[ -z "${NSS_PROPERTIES}" ] && NSS_PROPERTIES=/etc/blazegraph/RWStore.properties
```

Comment:
This file could be removed from this repo, if necessary.  I kept it in because it works on my laptop and it would work on any system that deploys the blazegraph outside of tomcat (I think it gets deployed this way by the debian package system and the redhat RPM system, not sure about the latter).